test/format-test.cc

```diff
@@ -1288,6 +1295,13 @@ TEST(format_test, format_infinity) {
   EXPECT_EQ("inf", fmt::format("{}", inf));
   EXPECT_EQ("+inf", fmt::format("{:+}", inf));
   EXPECT_EQ("-inf", fmt::format("{}", -inf));
+  EXPECT_EQ("  +inf", fmt::format("{:+06}", inf));
+  EXPECT_EQ("  -inf", fmt::format("{:+06}", -inf));
+   // '0'-fill option sets alignment to numeric overwriting any user-provided
+  // alignment
```

Comment:
Is this the desired behavior? Currently we lose the originally provided alignment because it seems to get replaced by `align::numeric` when using `0`-padding.

Reply:
Thanks, that works!